components/UploadForm.vue

```diff
@@ -245,6 +245,19 @@ const onFileUpload = async (event: Event) => {
             }
             if (fileRecord.fileType === 'application/epub+zip') {
               await processEPub({ buffer: fileBytes, file })
+            } else if (fileRecord.fileType?.startsWith('image/')) {
+              const coverMetadata = epubMetadataList.value[0]
```

Comment:
Should find a epub or pdf that does not have cover metadata, if all have metadata, consider no op

Reply:
[🎨 Add early check for cover existence](https://github.com/likecoin/nft-book-press/pull/354/commits/ac6ab4e0ea23fe98b1eec53b3568181cb949435a)

Reply:
[🎨 Use .find to prevent multiple cover images](https://github.com/likecoin/nft-book-press/pull/354/commits/862eaa0240fe4c6dea08563ee58be39fb76e1037)